src/Common/Commands.Common/AzureLongRunningJob.cs

```diff
@@ -191,11 +191,13 @@ public override string StatusMessage
                 returnValue.MyInvocation.BoundParameters.Add(parameter.Key, parameter.Value);
             }
 
-            foreach (var field in returnType.GetFields(BindingFlags.Instance | BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.DeclaredOnly))
+
+            foreach (var field in returnType.GetFields(BindingFlags.Instance | BindingFlags.Public | BindingFlags.NonPublic))
```

Comment:
I mistakenly changed this line in my PR - if you pull from my PR into release-5.1.0, this line is unchanged